compiler/testData/lineNumber/custom/functionCallWithLambdaParam.kt

```diff
@@ -13,3 +13,4 @@ fun foo(f: () -> Unit) {
 }
 
 // 2 6 9 12 13 3 4 7 8
+// IGNORE_BACKEND: JVM_IR
```

Comment:
Could you look into this test too: it's fails on Windows: https://teamcity.jetbrains.com/comparisonFailureDiffView.html?buildId=2450360&testId=453380
Maybe additional empty line is required to separate numbers from ignore directive

Reply:
It fails because of how `AbstractLineNumberTest#compareCustom` is implemented. The line number comment has to be the last (non-empty) line in the file for the non-IR test.

I've fixed it by swapping the comments for now, although it might make sense to change `AbstractLineNumberTest` in the future.